controllers/argocd/rolebinding.go

```diff
@@ -208,6 +208,11 @@ func getCustomRoleName(name string) string {
 	return ""
 }
 
+// Returns the name of the role for the supported namespaces for ArgoCDServer in the format of "SupportedNameSpace_SERVER_CLUSTER_ROLE"
+func getCustomRoleNameForSupportedNamespaces(name string, namespace string) string {
+	return fmt.Sprintf("%s_%s", namespace, common.ArgoCDServerClusterRoleEnvName)
```

Comment:
@ishitasequeira 
why are we appending SERVER_CLUSTER_ROLE to the name of this role? Sounds like it would be regular role and not actually a cluster role

Reply:
Updated the role name to be of the format `sourceNamespace_targetNamespace_argocd-server`